include/export.bash

```diff
@@ -0,0 +1,122 @@
+export-logs() {
+    declare desc="Exports and anonymizes logs for fault analysis. Usage: cbd export-logs [archive_name]"
+    declare archivename=$1
+    if [[ -z $archivename ]] || [[ "/" == $archivename ]]; then
+        archivename="cbd_export_$(date +%s)"
+        warn "Invalid archive filename. Generated filename: $archivename"
+    fi
+
+    local collectdir="collected_tmp"
```

Comment:
Would ve better to use a random temp dir instead of fixed one

Reply:
The Smartsense tool has to be able to find this directory to collect files from it. The tool cannot be parameterised as far as I know, so if I make this directory name random the tool will have no way of finding it.